cnf-certification-test/lifecycle/suite.go

```diff
@@ -372,3 +378,23 @@ func testPodsRecreation(env *provider.TestEnvironment) { //nolint:funlen
 		}
 	}
 }
+
+func testPodRequestsAndLimits(env *provider.TestEnvironment) {
+	var containersMissingRequestsOrLimits []string
+	ginkgo.By("Testing container resource requests and limits")
+
+	// Loop through the containers, looking for containers that are missing requests or limits.
+	// These need to be defined in order to pass.
+	for _, cut := range env.Containers {
+		if len(cut.Data.Resources.Limits) == 0 || len(cut.Data.Resources.Requests) == 0 {
```

Comment:
What is cut?

I think it is more clear to have two different conditions with two different messages. By the way one container could fail both conditions.

Reply:
This is true.  @mwlinca and I have been talking about this as well and I think there is more to add.  I'm going to make this work in progress as I fix.